events/EiffelActivityQueuedEvent/simple.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta": {
+    "domainId": "domainId-test",
+    "id": "acqueued-ec41-488e-93f2-fc2702b32f21",
+    "type": "EiffelActivityQueuedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "tags": [
+      "unit-test"
+    ],
+    "source": {
+      "host": "hostname",
+      "pid": 6132,
+      "name": "qwerty",
+      "serializer": {
```

Comment:
The serialilzer miss some mandatory fields. If we keep the serializer we should make a GAV that has values that looks real.